ament_cmake_test_isolated/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(ament_cmake_test_isolated)
+
+find_package(ament_cmake REQUIRED)
+
+install(
```

Comment:
@ivanpauno I don't need to `ament_export_dependencies(domain_coordinator)` here, right?  That's only necessary if it provides CMake stuff, yes?

Reply:
Yes